examples/vite/package.json

```diff
@@ -21,7 +21,7 @@
     "typescript": "^5.3.3",
     "vite": "^5.2.14",
     "vite-plugin-html": "^3.2.2",
-    "vite-plugin-node-polyfills": "^0.22.0"
+    "vite-plugin-node-polyfills": "0.17.0"
```

Comment:
Why backwards?

Reply:
18+ introduced an issue with shims, 17 was the last one without issue. I added a comment where its used that it must be 0.17.0

Reply:
Ok. Was 0.22 initially chosen for any other reason than "it was just the latest at the time" or can we rest assured that we don't need anything from it?

Reply:
this is the example app - we can update our recommendation for which version to use but consider it non-blocking of this specific PR

Reply:
@arielmelendez yeah i think 22 was just latest. I ran into this issue when working with other apps - we use this dep in a few different spots, can't remember where I first found the fix.